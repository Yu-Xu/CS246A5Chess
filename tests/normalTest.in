game human human
move d2 d4
move b8 c6
move c2 c3
move f7 f5
move g1 f3
move e7 e6
move g2 g4
move d7 d6
move g4 f5
move d6 d5
move f5 e6
move f8 e7
move c1 g5
move a7 a5
move d1 d3
move e7 g5
move f3 g5
move d8 g5
move f2 f4
move g5 h5
move f1 g2
move g8 f6
