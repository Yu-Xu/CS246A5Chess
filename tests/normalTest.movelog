8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ P _ _
3 _ _ _ _ 
2 PPP_PPPP
1 RNBQKBNR
  abcdefgh
8 r_bqkbnr
7 pppppppp
6  _n_ _ _
5 _ _ _ _ 
4  _ P _ _
3 _ _ _ _ 
2 PPP_PPPP
1 RNBQKBNR
  abcdefgh
8 r_bqkbnr
7 pppppppp
6  _n_ _ _
5 _ _ _ _ 
4  _ P _ _
3 _ P _ _ 
2 PP _PPPP
1 RNBQKBNR
  abcdefgh
8 r_bqkbnr
7 ppppp pp
6  _n_ _ _
5 _ _ _p_ 
4  _ P _ _
3 _ P _ _ 
2 PP _PPPP
1 RNBQKBNR
  abcdefgh
8 r_bqkbnr
7 ppppp pp
6  _n_ _ _
5 _ _ _p_ 
4  _ P _ _
3 _ P _N_ 
2 PP _PPPP
1 RNBQKB_R
  abcdefgh
8 r_bqkbnr
7 pppp_ pp
6  _n_p_ _
5 _ _ _p_ 
4  _ P _ _
3 _ P _N_ 
2 PP _PPPP
1 RNBQKB_R
  abcdefgh
8 r_bqkbnr
7 pppp_ pp
6  _n_p_ _
5 _ _ _p_ 
4  _ P _P_
3 _ P _N_ 
2 PP _PP P
1 RNBQKB_R
  abcdefgh
8 r_bqkbnr
7 ppp _ pp
6  _npp_ _
5 _ _ _p_ 
4  _ P _P_
3 _ P _N_ 
2 PP _PP P
1 RNBQKB_R
  abcdefgh
8 r_bqkbnr
7 ppp _ pp
6  _npp_ _
5 _ _ _P_ 
4  _ P _ _
3 _ P _N_ 
2 PP _PP P
1 RNBQKB_R
  abcdefgh
8 r_bqkbnr
7 ppp _ pp
6  _n_p_ _
5 _ _p_P_ 
4  _ P _ _
3 _ P _N_ 
2 PP _PP P
1 RNBQKB_R
  abcdefgh
