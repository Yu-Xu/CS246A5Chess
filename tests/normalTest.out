8 r_b_k_ r
7 _pp _ pp
6  _n_Pn _
5 p _p_ _q
4  _ P P _
3 _ PQ_ _ 
2 PP _P_BP
1 RN_ K _R
